### PR TITLE
grass.pygrass: Fixed E402 in pygrass/utils

### PR DESCRIPTION
**Pull request by arohanajit**
Added `noqa` to imports at top in line with other fixes in the repo

### CODE REVIEW

**Comment by echoix** [comment]
@petrasovaa Is this the correct pattern here? I think so, but I am not certain enough to directly approve.

**Comment by echoix** [comment]
(Conflict in .flake8 solved)

**Comment by arohanajit** [comment]
> @petrasovaa Is this the correct pattern here? I think so, but I am not certain enough to directly approve.

@echoix I made this change according to comments in this PR which was of similar nature: https://github.com/OSGeo/grass/pull/4706#pullrequestreview-2444142578

File: python/grass/pygrass/utils.py [hunk 1]
```diff
@@ -5,13 +5,16 @@
 
 import grass.lib.gis as libgis
 
+# flake8: noqa: E402
 libgis.G_gisinit("")
 
 import grass.lib.raster as libraster
 from grass.lib.ctypes_preamble import String
 from grass.script import core as grasscore
 from grass.script import utils as grassutils
```
Comment (echoix) [inline python/grass/pygrass/utils.py:14]: If moving these two lines out of the flake8 exclusion (below), does isort give the same sorting?

**Review by echoix — COMMENTED** [review]
Ah I remember now. In that case, I might suggest to move two imports out of the exclusion, if isort is happy and is correct.

In both cases it's good and I would agree to merge anyways.

File: python/grass/pygrass/utils.py (hunk 1 above)
Comment (arohanajit) [reply python/grass/pygrass/utils.py]: I have moved both of them out of flake8, isort didnt change anything

**Pull request merged**